plugin/evm/header/block_gas_cost_test.go

```diff
@@ -19,40 +19,54 @@ import (
 func TestBlockGasCost(t *testing.T) {
 	tests := []struct {
 		name         string
+		beforeAP4    bool
```

Comment:
nit: could we do ap4 or ap4Timestamp instead?

Reply:
I did not want to increase the diff for this one, but sure.